### ISSUE TITLE
Warning about circular dependencies

### ISSUE THREAD

**Issue by ucarbehlul** [newrelic/node-newrelic-aws-sdk#121]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )
Getting Warning: Accessing non-existent property 'agent' of module exports inside circular dependency
at startup

## Description
```
ERROR] [2022-01-13T16:38:13Z] (node:88887) Warning: Accessing non-existent property 'agent' of module exports inside circular dependency
    at emitCircularRequireWarning (internal/modules/cjs/loader.js:650:11)
    at Object.get (internal/modules/cjs/loader.js:664:5)
    at Object.<anonymous> (/Users/xxx/dev/xxx/xxx-client/node_modules/@newrelic/aws-sdk/nr-hooks.js:9:43)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.wrappedLoad [as _load] (/Users/xxx/dev/xxx/xxx-client/node_modules/newrelic/lib/shimmer.js:307:24)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
```

Same for 
```
Warning: Accessing non-existent property 'shim' of module exports inside circular dependency
```

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT share sensitive information, whether personal, proprietary, or otherwise! )

## Expected Behavior
There should not be a warning about circular dependency

[NOTE]: # ( Tell us what you expected to happen. )

## [Troubleshooting](https://discuss.newrelic.com/t/troubleshooting-frameworks/108787) or [NR Diag](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/troubleshooting/new-relic-diagnostics) results

[NOTE]: # ( Provide any other relevant log data. )
[TIP]:  # ( Scrub logs and diagnostic information for sensitive information )

## Steps to Reproduce

[NOTE]: # ( Please be as specific as possible. )
[TIP]:  # ( Link a sample application that demonstrates the issue. )

## Your Environment
```
 node --version
v14.15.4
```

From npm ls:
│ └── newrelic@8.7.0
  └── @newrelic/aws-sdk@4.1.0

OS: Mac OS Catalina 10.5.7

 Darwin Kernel Version 19.6.0: Tue Oct 12 18:34:05 PDT 2021; root:xnu-6153.141.43~1/RELEASE_X86_64 x86_64 i386 MacBookPro9,2 Darwin

## Additional context

[TIP]:  # ( Add any other context about the problem here. For example, relevant community posts or support tickets. )

**Comment by bizob2828** [comment]
Thanks for logging this @ucarbehlul, we're sorry for the troubles and are working on a fix.  In the meantime, we have deprecated this version of the package.

**Comment by michaelgoin** [comment]
@ucarbehlul We've pushed a fix with v4.1.1 we believe should resolve this issue. Sorry again for the issue!

**Comment by ucarbehlul** [comment]
Thanks!
